editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2632,6 +2653,15 @@ void Node3DEditorViewport::scale_freelook_speed(real_t scale) {
 	surface->queue_redraw();
 }
 
+bool Node3DEditorViewport::_is_nav_modifier_pressed(const String &p_name) {
+	return _is_shortcut_empty(p_name) || Input::get_singleton()->is_action_pressed(p_name);
+}
+
+bool Node3DEditorViewport::_is_shortcut_empty(const String &p_name) {
+	Ref<Shortcut> check_shortcut = ED_GET_SHORTCUT(p_name);
+	return check_shortcut->get_events().is_empty();
```

Comment:
It can error but this isn't a dynamic check, p_name will only ever be what is put in node_3d_editor_plugin source code. Since that's the case I thought we wanted it to error, right? Since it would signify a typo on the dev side. Should I catch it and print some other error message instead?

Reply:
I'd say yes, as that would prevent a segmentation fault here, so instead use some:
```cpp
ERR_FAIL_COND_V_MSG(check_shortcut.is_null(), true, "...");
```

Reply:
Done, hopefully this works